Tools/docker/Dockerfile-jammy

```diff
@@ -0,0 +1,44 @@
+#
+# PX4 development environment
+#
+
+FROM ubuntu:22.04
+LABEL maintainer="Daniel Agar <daniel@agar.ca>"
+
+ENV DEBIAN_FRONTEND noninteractive
+ENV LANG C.UTF-8
+ENV LC_ALL C.UTF-8
+
+RUN apt-get update && apt-get -y --quiet --no-install-recommends install \
+    ca-certificates \
+    wget
+
+ARG git_ref=main
+RUN wget https://raw.githubusercontent.com/PX4/PX4-Autopilot/$git_ref/Tools/setup/ubuntu.sh
```

Comment:
Just copy this in? One of the motivations for moving this in tree was to keep all of this versioned together. It should be totally valid to jump back to an old branch and rebuild this (for whatever weird reason).

Reply:
Ah of course copy would be simpler! 🤦